webview-ui/src/services/grpc-client-base.ts

```diff
@@ -149,7 +148,6 @@ export function createGrpcClient<T extends ProtoService>(service: T): GrpcClient
 								// Convert JSON back to protobuf message
 								const responseType = method.responseType
 								const response = responseType.fromJSON(message.grpc_response.message)
-								console.log("[DEBUG] grpc-client sending response:", response)
```

Comment:
<img width="1113" alt="image" src="https://github.com/user-attachments/assets/87d173bb-308b-43ab-97c9-63a4e1e6d47f" />


Can we be super strict with this file by getting rid of all console logs in this in prod. And also add a note that reminds people to NOT do this again as that was such a big problem the last time.